packages/components/order-tracking/src/order-tracking.html

```diff
@@ -0,0 +1,108 @@
+<div class="order-tracking">
+
+    <oui-back-button data-on-click="$ctrl.onBackButtonClick()">
+        <span data-translate="order_tracking_title"
+              data-translate-values="{ orderId: $ctrl.orderId }"></span>
+    </oui-back-button>
+
+    <div class="text-center">
+        <oui-spinner data-ng-if="$ctrl.isLoading" size="l">
+        </oui-spinner>
+    </div>
+
+    <oui-message type="error" data-ng-if="$ctrl.error">
+        <p>
+            <strong data-translate="order_tracking_error"></strong>
+        </p>
+        <p>
+            <span data-ng-bind="$ctrl.error.message"></span>
+            <br />
```

Comment:
Use `br` only for the semantic meaning of `I need a line return`, for example when writing a poem or an adress

in this case, you can use paragraphs